blankshield.js

```diff
@@ -45,27 +46,28 @@
    * @param {string} [strWindowName]
    * @param {string} [strWindowFeatures]
    */
-  blankshield.open = function(strUrl, strWindowName) {
+  blankshield.open = function(strUrl, strWindowName, strWindowFeatures) {
     var child;
     
-    if (strWindowName && strWindowName !== '_blank') {
-      open.apply(window, arguments);
+    if (strWindowName && (strWindowName === '_top' || strWindowName === '_self' || strWindowName === '_parent')) {
+      return open.apply(window, arguments);
     } else if (!oldIE) {
-      iframeOpen(strUrl);
+      return iframeOpen(strUrl, strWindowName, strWindowFeatures);
```

Comment:
window.open returns a ref to new window, so I made blankshield.open to return it too - so that it could be used as a drop-in replacement which won't break existing code that uses window.open - especially important when blankshield.patch() is used